include/Graphs/SVFG.h

```diff
@@ -173,6 +173,11 @@ class SVFG : public VFG
         return getSVFGNode(getDef(pagNode));
     }
 
+    /// Return the corresponding SVFGNodes to a given llvm::Value.
+    /// return an empty list, if the no mapping is possible
+    std::set<const SVFGNode*> fromLLVMValue(const llvm::Value* value) const;
```

Comment:
please also remove the namespace here.

Reply:
Should I also change the function name to `fromValue` then?

Reply:
Yes, please.

Reply:
Could you please change this name so that I could merge?

Reply:
Was on vacation. See #497.